completed css canvas medical-Bo Mercado

rbomercado opened the pull request:
![canvas medical](https://user-images.githubusercontent.com/97006645/152094689-d96d4735-6584-4ac2-86de-a6b47bd1e782.gif)

codingcodymiller reviewed (changes requested):
Hey Bo! The button in the "Ready To Learn More?" section is too far to the right of the page as compared to the example, please revise and submit a new screenshot with the revision.

rbomercado commented:
positioned the ready to learn get started button more to the left 
![image](https://user-images.githubusercontent.com/97006645/152228786-4352a56f-5728-44d9-b562-e6c256f3f7b5.png)

codingcodymiller reviewed (changes requested):
Hey Bo! The font family used in the text over the hero image and in the "Ready to learn more?" sections do not match the example, please revise and submit a new screenshot with your revision.

rbomercado commented:
updated screenshot

![_C__Users_rbome_repos_c0122-code-solutions_css-canvas-medical_index html](https://user-images.githubusercontent.com/97006645/152854730-58c80af0-f191-4da9-bdcc-eb34d01a4bd7.png)

codingcodymiller commented on css-canvas-medical/index.html:
```diff
@@ -0,0 +1,93 @@
+        <p class="text-align-center">Physician-owned primary care practices across
+    the country have already made Canvas the fabric of their care.</p>
+      </div>
+      <div class="clin-marg row justify-evenly align-center">
+        <div class="col-4">
+          <p>Cascade Family Practice,Portland OR </p>
+          <p>Temescal Creek Medicine,Oakland CA </p>
+          <p>San Juan Healthcare,Friday Harbor WA </p>
+        </div>
+        <div class=" col-4">
+          <p> Wards Medical Clinic , Bonifay FL</p>
+          <p> Caribou Medical, Towanda PA</p>
+          <p> Bay Tree Family Care, Red Bay AL</p>
+        </div>
+      </div>
```
This `row` is a direct child of an element with the `row` class. You should never have a `row` immediately inside of another element with the `row` class. You should put this `row` into a `column` class element of some size.

codingcodymiller commented on css-canvas-medical/styles.css:
```diff
@@ -0,0 +1,307 @@
+  width: 100%;
+}
+
+.row5 {
```
There should not be variants of the `row` class. This styling doesn't seem related to the functionality of the `row` class anyway, you should rename this class to describe its purpose.

codingcodymiller commented on css-canvas-medical/styles.css:
```diff
@@ -0,0 +1,307 @@
+  margin-top: 30px;
+}
+
+.row-1 {
+  flex-direction: column;
+  width: 100%;
+  padding-top: 50px;
+  padding-left: 50px;
+}
```
There should not be variants of the `row` class. This styling doesn't seem related to the functionality of the `row` class anyway, you should rename this class to describe its purpose.

codingcodymiller commented on css-canvas-medical/styles.css:
```diff
@@ -0,0 +1,307 @@
+  border: none;
+}
+
+.row1 {
+  margin-right: 25px;
+}
```
There should not be variants of the `row` class. This styling doesn't seem related to the functionality of the `row` class anyway, you should rename this class to describe its purpose.

codingcodymiller commented on css-canvas-medical/styles.css:
```diff
@@ -0,0 +1,307 @@
+  font-weight: 600;
+}
+
+.row5a {
+  padding-top: 70px;
+  padding-bottom: 80px;
+  padding-right: 50px;
+}
```
There should not be variants of the `row` class. This styling doesn't seem related to the functionality of the `row` class anyway, you should rename this class to describe its purpose.

codingcodymiller commented on css-canvas-medical/styles.css:
```diff
@@ -0,0 +1,307 @@
+.col-10 {
+  max-width: 100%;
+}
+}
```
I think this is an extra curly brace

rbomercado commented:
![_C__Users_rbome_repos_c0122-code-solutions_css-canvas-medical_index html (1)](https://user-images.githubusercontent.com/97006645/153132489-05d6f4ba-fe50-4679-ace4-161ee5fe6c32.png)


updated class extra row class names to more specific approprtiate names revamped all the font names as well fixed row that was a direct child

thebearingedge commented on css-canvas-medical/styles.css:
```diff
@@ -0,0 +1,283 @@
+  font-weight: 550;
+}
+
+.font-avenir-normal {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+}
+
+.font-avenir-normal-small {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+}
+
+.font-avenir-normal-medium {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+}
```
If you apply your `font-family` to the `body` one time, you won't have to apply it to separate elements. Also, I'm pretty sure `font-style` is `normal` by default, so there's no need to add it.

thebearingedge reviewed (changes requested):
Ok, here's my recommended approach to these changes.

1. first, go through and remove all extra code that you don't need or isn't actually having any affect. Check every property and remove it if it makes no difference.
2. style the body font and heading fonts, remove extra font styling after that
3. stick with class selectors instead of id selectors.

rbomercado commented:
Went through and deleted classes/selector/properties/values that didn't do anything applied font to body and only kept font/font classes where font needed to be anything other than the standard font-avenir-normal. Deleted grid.css. Switched ids to classes.
![_C__Users_rbome_repos_c0122-code-solutions_css-canvas-medical_index html (2)](https://user-images.githubusercontent.com/97006645/153306901-6efc90e9-aed7-4f0a-8c9b-e349609af770.png)
